nimbus-ui/nimbusui/src/app/components/platform/form/elements/rich-text.component.ts

```diff
@@ -0,0 +1,191 @@
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'size'">
+                    <button class="ql-header" value="1" aria-label="Heading Size 1"></button>
+                    <button class="ql-header" value="2" aria-label="Heading Size 2"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'blockquote'">
+                    <button class="ql-blockquote" aria-label="Insert Blockquote"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'codeBlock'">
+                    <button class="ql-code-block" aria-label="Insert Code Block"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'list'">
+                    <button class="ql-list" value="ordered" aria-label="Ordered List"></button>
+                    <button class="ql-list" value="bullet" aria-label="Unordered List"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'indent'">
+                    <button class="ql-indent" value="-1" aria-label="Unindent"></button>
+                    <button class="ql-indent" value="+1" aria-label="Indent"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'direction'">
+                    <button class="ql-direction" value="rtl" type="button"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'align'">
+                    <select class="ql-align">
+                        <option selected></option>
+                        <option value="center"></option>
+                        <option value="right"></option>
+                        <option value="justify"></option>
+                    </select>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'link'">
+                    <button class="ql-link" aria-label="Insert Link"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'image'">
+                    <button class="ql-image" aria-label="Insert Image"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'video'">
+                    <button class="ql-video" aria-label="Insert Video"></button>
+                </ng-template>
+
+                <ng-template [ngIf]="toolbarFeature === 'clean'">
+                    <button class="ql-clean" aria-label="Remove Styles"></button>
+                </ng-template>
+
+            </ng-template>
+        </p-header>
+        
+    </p-editor>
+   `
+})
+export class RichText extends BaseControl<String> {
+
+    @ViewChild(NgModel) model: NgModel;
+
+    constructor(wcs: WebContentSvc, controlService: ControlSubscribers, cd: ChangeDetectorRef) {
+        super(controlService, wcs, cd);
+    }
+
+    public get placeholder(): string {
```

Comment:
Can use in template with safe operator for null check

Reply:
Fixed.